projects/components/src/datagrid/datagrid.component.spec.ts

```diff
@@ -140,24 +134,22 @@ describe('DatagridComponent', () => {
 
             it('has multi selection capabilities when set to multi', function(this: HasFinderAndGrid): void {
                 this.finder.hostComponent.selectionType = GridSelectionType.Multi;
-                let rowsEmitted = 0;
-                this.finder.hostComponent.selectionChanged = (selection: MockRecord[]) =>
-                    (rowsEmitted = selection.length);
                 this.finder.detectChanges();
                 expect(this.clrGridWidget.getSelectionType()).toBe(GridSelectionType.Multi);
+                this.clrGridWidget.selectRow(0);
+                expect(this.finder.hostComponent.getSelection()).toEqual([mockData[0]]);
```

Comment:
Why do we get an array, and then a single object?

Reply:
First one is also an arrray.